packages/frontier-plugins/plugin-qa/src/commands/qa/create-screen-object/index.ts

```diff
@@ -3,7 +3,7 @@ import path from 'path'
 import {Command, flags} from '@oclif/command'
 import { CONSTANTS } from '../../../utils/constants'
 import { copyScreenObjectFile, updateScreenObjectFile, updateFileName } from '../../../utils/files'
-import { checkIfProjectIsValid } from '../../../utils/utilities'
+import { checkIfProjectIsValid } from '../../../utils/utilities' 
```

Comment:
typescript has a syntax whe you can alias directories, are you familiar with them?